tool/setup.sh

```diff
@@ -2,16 +2,16 @@
 set -euxo pipefail
 cd "$(dirname "$0")/.."
 
-for package in $(yq -r ".dev_dependencies | keys | .[]" pubspec.yaml); do
-  version="$(yq -r ".dev_dependencies.$package" pubspec.yaml)"
-  dart pub global activate "$package" "$version"
-done
+dart pub global activate coverage
+dart pub global activate custom_lint
+dart pub global activate fvm
+dart pub global activate melos
```

Comment:
I'm not a big fan of this.
Maybe we should first migrate this repo to a nix flake before we merge this one

Reply:
I will pin them to exact versions for now, but yes the flake will be better than this.
Unfortunately it needs more work so I would like to have this PR merged without waiting for it.